src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -174,6 +198,13 @@ public static void teardown() throws Exception {
     public void after() {
         ComputeEngineCloud cloud = (ComputeEngineCloud) r.jenkins.clouds.get(0);
         cloud.configurations.clear();
+
+
+        try {
+            client.getCompute().instanceTemplates().delete(cloud.projectId, nameFromSelfLink(TEMPLATE)).execute();
```

Comment:
This breaks the encapsulation the client wrapper provides. It would be preferred to add an instanceDelete method to the wrapper.